go/client/cmd_log_send.go

```diff
@@ -205,6 +209,7 @@ func (c *CmdLogSend) ParseArgv(ctx *cli.Context) error {
 	}
 	c.noConfirm = ctx.Bool("no-confirm")
 	c.numBytes = ctx.Int("n")
+	c.feedback = ctx.String("feedback")
```

Comment:
shouldn't `getFeedback` be modified to skip prompted if this is provided?

Reply:
The idea is that you could still get interactivity with some prefixed note. Probably would never get used so I can remove it.

Reply:
should just skip `getFeedback` if this is set